### PR TITLE
Jop/crown

### PR DESCRIPTION
**Pull request by joannapierquin**

### CODE REVIEW

File: adapted-articles/C/Crown.xml [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
-<p>Jesus’ crown of thorns was a circlet formed from a prickly shrub—an ironic parody of a victor’s wreath (<a href="?bref=Mark.15.17-18">Mk 15:17-18</a>). Its combination with the robe, scepter (<a href="?bref=Matt.27.27-29">Mt 27:27-29</a>), and satirical inscription on the cross that Jesus was “the King of the Jews” (<a href="?bref=Mark.15.26">Mk 15:26</a>), were all meant to mock him as a defeated messianic aspirant.</p>
+<p>In the NT the most common word for “crown” means a laurel crown worn at banquets or a prize given as a civic or military honor. The apostle Paul used it as an athletic prize when he urged Christians to strive for a “crown” that would not weaken (<a href="?bref=1Cor.9.25">1 Cor 9:25</a>; <a href="?bref=2Tim.2.5">2 Tm 2:5</a>). Paul regarded his converts as his “joy and crown” (<a href="?bref=Phil.4.1">Phil 4:1</a>; <a href="?bref=1Thes.2.19">1 Thes 2:19</a>).</p>
```
Comment (JonathanS77) [inline adapted-articles/C/Crown.xml]: - Not sure "Paul used it as an atheltic prize" has the same meaning as "alluded to its use as an athletic prize".

Comment (JonathanS77) [reply adapted-articles/C/Crown.xml]: - Wither I= weaken? Wither = se gâter, se flétrir

File: adapted-articles/C/Crown.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<p>A victor’s crown symbolizes the eternal life inherited by Christians who have persevered (<a href="?bref=Jas.1.12">Jas 1:12</a>; <a href="?bref=1Pet.5.4">1 Pt 5:4</a>; <a href="?bref=Rev.2.10">Rv 2:10</a>; <a href="?bref=Rev.3.11">3:11</a>). In the book of Revelation the victories of the locusts (<a href="?bref=Rev.9.7">9:7</a>), the woman (<a href="?bref=Rev.12.1">12:1</a>), and Christ (<a href="?bref=Rev.6.2">6:2</a>; <a href="?bref=Rev.14.14">14:14</a>) are symbolized by laurel crowns. A different Greek word, meaning a royal crown, is used for the diadems on the heads of the dragon (<a href="?bref=Rev.12.3">12:3</a>), the beast from the sea (<a href="?bref=Rev.13.1">13:1</a>), and Christ (<a href="?bref=Rev.19.12">19:12</a>).</p>
+<p>Jesus’ crown of thorns was a small circle formed from a bush covered in prickles—an ironic parody of a victor’s crown (<a href="?bref=Mark.15.17-18">Mk 15:17-18</a>). Its combination with the robe, scepter (<a href="?bref=Matt.27.27-29">Mt 27:27-29</a>), and satirical inscription on the cross that Jesus was “the King of the Jews” (<a href="?bref=Mark.15.26">Mk 15:26</a>), were all meant to mock him as a defeated messianic aspirant.</p>
```
Comment (jgonz3) [inline adapted-articles/C/Crown.xml]: It was decided for good or for ill to expand the possession mark on people’s names ending in “s” like so: “Jesus’s.”

cf. General Style Guide.md

File: changelog/C/Crown.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# What Has Changed In the “Crown” Article
+
+## Major changes
```
Comment (jgonz3) [inline changelog/C/Crown.md:3]: Same comment as the one made on the Shepherd article: The changes here seem sensible and need not be repeated in the changelog. An approximate count of the number of changes would be sufficient (no need to rework this file, you can do so on the next articles.)

File: changelog/C/Crown.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+The following words or expressions were changed because they seemed too complicated to the general understanding of the target audience:
+- a narrow band of silk studded with jewels --> a narrow band of silk covered with jewels
+- wreath --> crown
```
Comment (jgonz3) [inline changelog/C/Crown.md:6]: A handful of other articles use the word “wreath” as well, perhaps if this change becomes systematic we can add it to the replacement table in the General Style Guide.md

**Review by jgonz3 — DISMISSED** [review]
Good job!

One change requested: Please see my comment on the possession mark :)

**Review by mvahowe — APPROVED** [review]
LGTM

**Pull request merged**